### PR TITLE
fix(public-cloud): fix issue payment method checkbox

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/TAPC-83` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | TAPC-150 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I fixed issue payment method checkbox

### CODE REVIEW

**Review by qpavy — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/new/payment/index.html [hunk 1]
```diff
@@ -55,181 +55,188 @@ <h1 data-translate="pci_project_new_payment_title"></h1>
...
-                        data-ng-if="$ctrl.model.paymentMethod.paymentType === 'CREDIT'"
+                    <!--Credit card and PayPal-->
+                    <ovh-payment-method-integration
+                        data-callback-status-param-url-name="paymentStatus"
+                        data-payment-method-type="$ctrl.model.paymentMethod"
+                        data-on-initialized="$ctrl.onIntegrationInitialized.bind($ctrl)"
+                        data-on-submit="$ctrl.onIntegrationSubmit.bind($ctrl)"
+                        data-on-submit-error="$ctrl.onIntegrationSubmitError.bind($ctrl)"
+                        data-on-submit-success="$ctrl.onIntegrationSubmitSuccess.bind($ctrl)"
+                        data-ng-if="$ctrl.displayCreateProjectButton()"
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/new/payment/index.html:80]: I have a question for you for this check in your parent PR.

Comment (sachinrameshn) [reply packages/manager/modules/pci/src/projects/new/payment/index.html:80]: https://github.com/ovh/manager/pull/9847

Comment (brofik) [reply packages/manager/modules/pci/src/projects/new/payment/index.html:80]: Ok, I'll check it in the parent PR.
Beside this, are you Ok with the review of this PR so I can merge with its parent ?

Comment (sachinrameshn) [reply packages/manager/modules/pci/src/projects/new/payment/index.html:80]: yes good Rafik

**Review by sachinrameshn — APPROVED** [review]

**Pull request merged**